lib/slax/slack.ex

```diff
@@ -129,4 +132,86 @@ defmodule Slax.Slack do
         :ok
     end
   end
+
+  @doc """
+    Opens a modal view
+  """
+  def open_modal(%{trigger_id: trigger_id, view: view}) do
+    request =
+      Jason.encode!(%{
+        trigger_id: trigger_id,
+        view: view
+      })
+
+    case Http.post(
+           "#{api_url()}/views.open",
+           request,
+           "Content-Type": "application/json",
+           Authorization: "Bearer #{api_token()}"
+         ) do
+      {:ok, %{body: %{"ok" => false, "error" => error}}} ->
+        IO.puts("Error for #{trigger_id}: #{error}")
```

Comment:
We probably should `require Logger` up top and then use that here instead of `IO.puts`